1001-2000/LIT1512GadlaY.xml

```diff
@@ -47,6 +48,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+         <creation/>
+         <abstract>
+            <p>Collection of hagiographic texts in honour of <persName ref="PRS10245Yared">Yared</persName>.</p>
```

Comment:
We should disambiguate this record and the record LIT1303Dersan 

So record LIT1303Dersan  should have the title 
```
<title xml:lang="gez" xml:id="t1">ድርሳን፡ ወገድል፡ ዘያሬድ፡</title>
            <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsān wa-gadl za-Yāred</title>
            <title xml:lang="en" corresp="#t1">Homily and Vita of Yāred</title>
```
which is one text, and  LIT1512GadlaY

should have as t1 Hagiographic Dossier in Honour of St Yared

malk is also its part